CMakeLists.txt

```diff
@@ -108,6 +108,9 @@ set(action_files
   "action/MoveGroupSequence.action"
   "action/Pickup.action"
   "action/Place.action"
+  "action/LocalPlanner.action"
+  "action/GlobalPlanner.action"
+  "action/HybridPlanning.action"
```

Comment:
Only a tiny comment, this is `*Planner`, `*Planner`, `*Planning`, maybe the last should be `action/HybridPlanner`?